### DISCUSSION TITLE
Email automation

### DISCUSSION THREAD

**Discussion by takkbbl** [Glench/ExtPay#132] [Q&A]
Hi guys, I may need your help in terms of email automation. I'd like to send automated emails when someone signs up, pays for a subscription or when the trial expires. Are there any best practices on how to do this? I'm familiar with Python, so that wouldn't be a problem for me. Thanks for your support!

**Comment by pizzaisdavid** [comment]
@takkbbl did you find an answer for this? It seems stripe has some built in email notifications

**Reply by takkbbl** [reply]
> @takkbbl did you find an answer for this? It seems stripe has some built in email notifications

Hi, with Stripe I can automate emails once the user purchased but I'm still struggling to find a solution to automate emails before a user makes a purchase. For example to automate new sign ups to the trial period or when trial period is about to end.
